easybuild/easyconfigs/q/Qt6/Qt6-6.6.3_riscv.patch

```diff
@@ -0,0 +1,11 @@
+--- qtwebengine/src/3rdparty/chromium/base/allocator/partition_allocator/partition_alloc.gni.orig	2025-02-03 16:03:22.374870122 +0100
```

Comment:
@julianmorillo Please add a short descirption on top + mention author or source of the patch.

This goes for all 3 patch files.

Also, don't forget to update the checksums accordingly